Trimble.Modus.Components/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+﻿using Trimble.Modus.Components;
+
+[assembly: Preserve]
+[assembly: XmlnsDefinition("http://modus.trimble.com/controls", "Trimble.Modus.Components")]
```

Comment:
Can we have it as http://modus.trimble.com/components?

Reply:
Updated the url